app/views/evaluation/edit.html.haml

```diff
@@ -23,9 +24,15 @@
     = render partial: 'evaluation/enrollment.html.haml', :locals => {:f => f}
 
     #instructor_select_wrapper
-    %label{for: "evaluation_instructor", class: "col-sm-2 control-label"} Instructor
-    .col-sm-4
-      = f.text_field :instructor, class: "form-control", placeholder: "e.g. Tiffani Williams", value: @evaluation.instructor.try(:name)
+      %label{for: "evaluation_instructor_id", class: "col-sm-2 control-label"} Instructor
+      .col-sm-4
+        = f.select "instructor_id", @instructors, {}, class: "form-control"
```

Comment:
Could you suggest me how to set :instructor object to selected option? 
instructor=>"" always